public/js/actions/sources.js

```diff
@@ -337,7 +316,7 @@ function getTextForSources(actors) {
       if (pending.size == 0) {
         // Sort the fetched sources alphabetically by their url.
         deferred.resolve(
-          fetched.sort(([aFirst], [aSecond]) => aFirst > aSecond));
+          fetched.sort(([aFirst], [aSecond]) => aFirst > aSecond ? -1 : 1));
```

Comment:
we'll likely want to revisit this when we add full project search, but that is a little ways off.